README.md

```diff
@@ -28,18 +28,77 @@ python manage.py db upgrade
 python run.py
 ```
 
-## v0 API endpoints available
+## Available v0 API endpoints
 
-| API endpoint                  | description                                                                |
-| ------------                  | -----------                                                                |
-| /api/v0/ros/systems           | Shows list of all systems from Host Inventory having a Performance Profile |
-| /api/v0/ros/systems/<host_id> | Shows the Performance Profile of the selected <host_id>                    |
-| /api/v0/ros/status            | Shows the status of the server                                             |
+### Request
+`GET /api/ros/v0/systems` Shows list of all systems from Host Inventory having a Performance Profile
 
-To run via API:
-```
-curl -v -H "x-rh-identity: <identity-header-value>" -H "Content-Type: application/json" http://localhost:8080/api/ros/v0/systems
-curl -v -H "x-rh-identity: <identity-header-value>" -H "Content-Type: application/json" http://localhost:8080/api/ros/v0/status
-curl -v -H "x-rh-identity: <identity-header-value>" -H "Content-Type: application/json" http://localhost:8080/api/ros/v0/systems/<host_id>
-```
-For local dev setup, please note that use the x-rh-identity header encoded from your account number, the one used while running `make insights-upload-data` and `make ros-upload-data` commands.
+    curl -v -H "x-rh-identity: <header_value>" -H "Content-Type: application/json" https://ci.cloud.redhat.com/api/ros/v0/systems
+
+### Response
+
+    HTTP/1.1 200 OK
+    Date: Thu, 24 Feb 2011 12:36:30 GMT
+    Status: 200 OK
+    Connection: close
+    Content-Type: application/json
+    Content-Length: 2
```

Comment:
I think we should remove this response header from the output/readme file.

Reply:
@patilsuraj767 I guess there should be no harm in showing the response header. For reference: https://github.com/bbc/REST-API-example/blob/master/README.md#create-a-new-thing. We can change it to precisely what details are shown in our response header. Thoughts?
@gnurag comments?

Reply:
Ok, we can keep the response header.